Bruin Bite/View Controllers/MenuVC.swift

```diff
@@ -77,7 +77,7 @@ class MenuVC: UIViewController, UICollectionViewDelegate, UICollectionViewDataSo
         case 21..<24:
             self.topBar.timeSelected("Night")
         default:
-            print("Current Meal Period could not be determined")
+            Logger.shared.handle(type: .error, message: "Current Meal Period could not be determined")
```

Comment:
Can you reconfirm if this needs to be `.error` or if we can change it to `.warning` (read the aforementioned descriptions for the intention of each type, basically, if the meal period not being determined breaks things in a very noticeable fashion then leave it as `.error` but if not, and its a minor inconvenience, then change to `warning`)

Reply:
I believe this wold result in the behavior we were seeing last year where the menu would not load until you clicked on a different time. However, this switch statement covers all possible values of hour from zero (inclusive) to 24 (exclusive). Therefore the error is more of a formality. I can switch it to warning though.